tests/foreman/installer/test_installer.py

```diff
@@ -1345,112 +1344,107 @@
+@upgrade
+@tier1
+def test_positive_foreman_module():
+    """Check if SELinux foreman module has the right version
 
-    @upgrade
-    @tier1
-    def test_positive_check_installer_services(self):
-        """Check if services start correctly
+    :id: a0736b3a-3d42-4a09-a11a-28c1d58214a5
 
-        :id: 85fd4388-6d94-42f5-bed2-24be38e9f104
+    :expectedresults: Foreman RPM and SELinux module versions match
+    """
+    rpm_result = ssh.command('rpm -q foreman-selinux')
+    assert rpm_result.return_code == 0
 
-        :expectedresults: All services {'elasticsearch', 'foreman-proxy', 'httpd',
+    semodule_result = ssh.command('semodule -l | grep foreman')
+    assert semodule_result.return_code == 0
 
-        'rh-mongodb34-mongod', 'postgresql', 'pulp_celerybeat', 'pulp_resource_manager',
+    # Sample rpm output: foreman-selinux-1.7.2.8-1.el7sat.noarch
+    version_regex = re.compile(r'((\d\.?)+[-.]\d)')
+    rpm_version = version_regex.search(''.join(rpm_result.stdout)).group(1)
+    # Sample semodule output: foreman        1.7.2.8
+    semodule_version = version_regex.search(''.join(semodule_result.stdout)).group(1)
+    rpm_version = rpm_version[:-2]
+    assert rpm_version.replace('-', '.') == semodule_version
 
-        'pulp_workers', 'qdrouterd', 'qpidd', 'tomcat'} are started
-        """
-        major_version = get_host_info()[1]
-        services = (
-            'dynflow-sidekiq@orchestrator',
-            'dynflow-sidekiq@worker',
-            'dynflow-sidekiq@worker-hosts-queue',
-            'foreman-proxy',
-            'httpd',
-            'rh-mongodb34-mongod',
-            'postgresql',
-            'pulp_celerybeat',
-            'pulp_resource_manager',
-            'pulp_streamer',
-            'pulp_workers',
-            'puppetserver',
-            'qdrouterd',
-            'qpidd',
-            'rh-redis5-redis',
-            'smart_proxy_dynflow_core',
-            'squid',
-            'tomcat6' if major_version == RHEL_6_MAJOR_VERSION else 'tomcat',
-        )
 
-        # check `services` status using service command
-        if major_version >= RHEL_7_MAJOR_VERSION:
-            status_format = 'systemctl status {0}'
-        else:
-            status_format = 'service {0} status'
+@upgrade
+@tier1
+def test_positive_check_installer_services():
+    """Check if services start correctly
 
-        for service in services:
-            with self.subTest(service):
-                result = ssh.command(status_format.format(service))
-                self.assertEqual(result.return_code, 0)
-                self.assertEqual(len(result.stderr), 0)
+    :id: 85fd4388-6d94-42f5-bed2-24be38e9f104
 
-        # check status reported by hammer ping command
-        result = ssh.command(
-            'hammer -u {0[0]} -p {0[1]} ping'.format(settings.server.get_credentials())
-        )
+    :expectedresults: All services {'elasticsearch', 'foreman-proxy', 'httpd',
```

Comment:
Given that this list of services in docstring is already outdated, would it make sense to remove it? Just say "All services are started".
Non-blocking comment.

Reply:
you are right, "All services are started" is looks more appropriate than mentioning each service in the docstring.